### PR TITLE
Add colored disk usage bars to display warning and error condition wh…

### LINKED ISSUE #350: the drive color changes if it is almost full
I hope the drive color changes if it is almost full
screenshot from 2018-03-05 18 23 44

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/55807244-the-drive-color-changes-if-it-is-almost-full?utm_campaign=plugin&utm_content=tracker%2F65602118&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F65602118&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by CodingSpiderFox**
…en disk becomes too full

Fixes #350

### CODE REVIEW

**Comment by CodingSpiderFox** [comment]
PS: on my fork, you can find a different design 
https://github.com/CodingSpiderFox/files/tree/feature/bluediskusagebars
image

**Comment by CodingSpiderFox** [comment]
This PR makes it look like this

When >= 90%: 
image

When >= 75%:
image

File: src/View/Widgets/DiskRenderer.vala [hunk 1]
```diff
@@ -69,8 +69,19 @@ public class Marlin.CellRendererDisk : Gtk.CellRendererText {
...
         /* Filled part of bar */
-        context.add_class ("fill-block");
+        double filled_percent = ((double) disk_size - (double) free_space) / (double) disk_size;
+        if (filled_percent >= 0.9) {
+            context.add_class ("fill-block-critical");
+        }
+        else if (filled_percent >= 0.75) {
+            context.add_class ("fill-block-warn");
+        }
+        else {
```
Comment (jeremypw) [inline src/View/Widgets/DiskRenderer.vala:81]: Elementary code style: else follows brace on same line.   `} else if () {`

Comment (CodingSpiderFox) [reply src/View/Widgets/DiskRenderer.vala:81]: > @CodingSpiderFox Thanks for your first contribution! I agree that the colors are more distinct with a slightly thicker bar and I am happy for the bar height to increase to 4px. I find 5px a little too thick for my liking. I think any more change to the appearance would require prior approval of the UX team so perhaps would be best raised as a separate issue/PR where details of colors etc can be fully debated.
> 
> There is a minor comment inline re elementary code style

So should I add the 4px to this PR or open another?

Comment (jeremypw) [reply src/View/Widgets/DiskRenderer.vala:81]: I am happy for it to be included here.

Comment (CodingSpiderFox) [reply src/View/Widgets/DiskRenderer.vala]: Done

**Pull request merged**